examples/proc_GDS_pulse_capture.py

```diff
@@ -0,0 +1,91 @@
+r"""
+Calculate actual beta as a function of pulse length
+===================================================
+Assuming the data is the capture of the pulse sequence as seen on the GDS
+oscilloscope, here the data is converted to analytic power, frequency filtered
+and the absolute is taken prior to integrating to return the beta where
+:math:`\beta = \int \sqrt{P(t)} dt` 
+"""
+import pyspecdata as psd
+import matplotlib.pyplot as plt
+from numpy import sqrt
+
+atten_ratio = 101.35  # attenutation ratio
+show_all = False
+with psd.figlist_var() as fl:
+    for filename, nodename, label in [
+        ("240730_test_amp1_fin_pulse_calib.h5", "pulse_calib_1", "amplitude = 1"),
+        ("240730_test_amp0p1_fin_pulse_calib.h5", "pulse_calib_1", "amplitude = 0.1"),
+    ]:
+        d = psd.find_file(
+            filename, expno=nodename, exp_type="ODNP_NMR_comp/test_equipment"
+        )
+        beta = psd.ndshape(d).pop("t").alloc().rename("p_90", "desired_beta")
+        beta.setaxis("desired_beta", d.get_prop("desired_betas"))
+        t_v_beta = psd.ndshape(d).pop("t").alloc().rename("p_90", "At_p")
```

Comment:
I don't understand what this is supposed to be, and I don't think that it's necessary